networkx/generators/classic.py

```diff
@@ -322,7 +322,7 @@ def grid_2d_graph(m,n,periodic=False,create_using=None):
         boundary nodes via periodic boundary conditions.
     """
     G=empty_graph(0,create_using)
-    G.name="grid_2d_graph"
+    G.name="grid_2d_graph(%d,%d)"%(m,n)
```

Comment:
Just to make sure before I do this - is there a clear consensus that `str.format` is going to be standard for networkx going forward?  I used `%` to be consistent with the other lines in the file.  If it's going to be the new standard for networkx, then I'll update the whole file.